public/app/core/utils/explore.ts

```diff
@@ -184,6 +181,20 @@ export const safeParseJson = (text: string) => {
   }
 };
 
+export const safeStringifyValue = (value: any, space?: number) => {
```

Comment:
This doesn't seem to be used?

Reply:
Good catch, it should be used when parsing errors. I'll take a look.